azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -63,6 +63,12 @@ def _translate_input(self, circuit, data_format, input_params):
         else:
             return super()._translate_input(circuit, data_format, input_params)
 
+    def _set_gateset(self, **kwargs):
+        self._gateset = kwargs.get("gateset", "qis")
```

Comment:
```suggestion
        self._gateset = GETSET_MAP[kwargs.get("gateset", "qis")]
```

Reply:
We discussed this offline, and we're keeping IonQ's convention.